bigtable/tests/unit/test_cluster.py

```diff
@@ -64,8 +65,8 @@ def test_constructor_non_default(self):
         instance = _Instance(self.INSTANCE_ID, client)
 
         cluster = self._make_one(self.CLUSTER_ID, instance,
-                                 serve_nodes=SERVE_NODES)
-        self.assertEqual(cluster.cluster_id, self.CLUSTER_ID)
+                                 self.LOCATION_ID, serve_nodes=SERVE_NODES)
+        self.assertEqual(cluster.cluster_id, self.CLUSTER_ID, self.LOCATION_ID)
```

Comment:
That third argument doesn't look right (it would replace the standard assertion error message).

Reply:
Removed self.LOCATION_ID, thanks.